apis/v1alpha1/shared_types.go

```diff
@@ -173,31 +173,34 @@ type RouteGatewayStatus struct {
 	// GatewayRef is a reference to a Gateway object that is associated with
 	// the route.
 	GatewayRef GatewayReference `json:"gatewayRef"`
+
 	// Conditions describes the status of the route with respect to the
-	// Gateway.  For example, the "Admitted" condition indicates whether the
-	// route has been admitted or rejected by the Gateway, and why.  Note
-	// that the route's availability is also subject to the Gateway's own
-	// status conditions and listener status.
+	// Gateway. The "Admitted" condition must always be specified by controllers
+	// to indicate whether the route has been admitted or rejected by the Gateway,
+	// and why. Note that the route's availability is also subject to the Gateway's
+	// own status conditions and listener status.
 	//
 	// +listType=map
 	// +listMapKey=type
+	// +kubebuilder:validation:MinItems=1
```

Comment:
I think this is technically a breaking change. Can we file a separate issue/PR to track this for v1alpha2? Admittedly this is super nitpicky and unlikely to actually break anyone at this point, but good to be as consistent as possible with backwards compatibility.

Reply:
@robscott done, xref: https://github.com/kubernetes-sigs/service-apis/issues/542